.mci.yml

```diff
@@ -78,8 +78,9 @@ tasks:
           params:
               working_dir: "mongo-cxx-driver/build"
               script: |
-                  PKG_CONFIG_PATH="/data/tmp/c-driver-install/lib/pkgconfig" /opt/cmake/bin/cmake -DBUILD_UNIT_TESTS=true -DBUILD_EXAMPLES=true -DCMAKE_BUILD_TYPE=${build_type} CMAKE_INSTALL_PREFIX=/data/tmp/cxx-driver-install ..
+                  PKG_CONFIG_PATH="/data/tmp/c-driver-install/lib/pkgconfig" /opt/cmake/bin/cmake -DBUILD_EXAMPLES=true -DCMAKE_BUILD_TYPE=${build_type} CMAKE_INSTALL_PREFIX=/data/tmp/cxx-driver-install ..
```

Comment:
I think we do because the examples rely on the driver being installed, and the default install prefix is /usr/local. We could change the default install prefix, but that may be unexpected for users used to a typical `make && sudo make install` workflow. Alternatively we could make the examples not depend on the installed driver, but that would lessen their value as they are our main test that the installation process actually works.